app/views/layouts/_header.html.erb

```diff
@@ -112,19 +112,25 @@
   <div class="modal-dialog modal-lg modal-dialog-scrollable">
     <div class="modal-content">
       <div class="modal-header">
-        <h1 class="modal-title fs-5" id="select_language_dialog_label"><%= t ".select_language" %></h1>
+        <h1 class="modal-title fs-5" id="select_language_dialog_label"><%= t "languages_panes.select_language_list.header.select_language" %></h1>
         <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="<%= t "javascripts.close" %>"></button>
       </div>
       <div class="modal-body px-1">
-        <% if current_user&.id %>
-          <%= form_tag basic_preferences_path, :method => "PUT" do %>
-            <%= hidden_field_tag "referer", request.fullpath %>
-            <%= hidden_field_tag "language", I18n.locale %>
-            <%= render "layouts/select_language_list" %>
-          <% end %>
-        <% else %>
-          <%= render "layouts/select_language_list" %>
-        <% end %>
+        <div class="mb-3 px-2">
+          <%= text_field_tag :language_search,
+                             params[:language_search],
+                             :placeholder => t("languages_panes.select_language_list.search_language", :default => "Search..."),
```

Comment:
Same question here about the scope of the resource string, though obviously in this case it depends on if the form stays here or moves but in general we put translations in the scope where they're used so we can refer to them with a relative name.

Reply:
>  it depends on if the form stays here

Right, I wanted to keep all translations in the same place, also anticipating that we will move the form again.